test/test_dictionary/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+add_executable(testdictionary
+        $<TARGET_OBJECTS:libgerbera>
```

Comment:
Adding reference to the `libgerbera` defined in the parent **CMakeLists.txt**.  This allows any tests within here to utilize the library.